Cargo.lock

```diff
@@ -23,6 +23,17 @@ version = "1.2.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "aae1277d39aeec15cb388266ecc24b11c80469deae6067e17a1a7aa9e5c1f234"
 
+[[package]]
```

Comment:
I don't like this big diff in Cargo.lock, but I guess faster CI comes with this cost.